tools/dashboard-plugin-tests/cypress/integration/test-plugin/layout.spec.ts

```diff
@@ -0,0 +1,22 @@
+// (C) 2019-2022 GoodData Corporation
+import * as Md from "../../../md/full";
+import { withTestConfig } from "../../tools/configure";
+
+import { newTestPlugin, newTestPluginLink } from "../../support/plugins";
+
+describe("Layout", () => {
+    beforeEach(() => {
```

Comment:
This beforeEach block is repeated quite often, as a follow up we might think about how to make it less repetitive maybe?